packages/collector/src/metrics/pid.js

```diff
@@ -5,13 +5,21 @@
 
 'use strict';
 
-const pidStore = require('../pidStore');
+let pid;
 
-exports.payloadPrefix = 'pid';
-// @ts-ignore - Cannot redeclare exported variable
-exports.currentPayload = pidStore.pid;
+/**
+ * @param {any} config
```

Comment:
```suggestion
 * @param {import('@instana/core/src/metrics').InstanaConfig} config
```

its always better to use the correct type

Reply:
Good catch! But the path is wrong. I cannot use your suggestion. I will update manually.

Reply:
Done https://github.com/instana/nodejs/pull/1556/commits/bcf51e9f64f5f38176b10a552cc774933fb744d0